skimage/feature/_canny.py

```diff
@@ -205,6 +212,10 @@ def canny(image, sigma=1., low_threshold=None, high_threshold=None,
         quantiles of the edge magnitude image, rather than absolute
         edge magnitude values. If ``True`` then the thresholds must be
         in the range [0, 1].
+    mode : str, {'reflect', 'constant', 'nearest', 'mirror', 'wrap'}
+        The ``mode`` parameter determines how the array borders are
+        handled during Gaussian filtering, where ``cval`` is the value when
```

Comment:
Should we add the `cval` argument also? It must then be passed to the `_preprocess` function.

Reply:
I went ahead and added it